Ladybird/AppKit/UI/Tab.mm

```diff
@@ -68,6 +68,9 @@ - (instancetype)init
                               backing:NSBackingStoreBuffered
                                 defer:NO];
 
+    // Remember last window position
+    self.frameAutosaveName = @"window";
```

Comment:
Postmortem, but this belongs inside the `if (self)` block.

Reply:
Good point, I didn't saw that for some weird reason. Do you want to create a fix commit, or you want me to create one?

Reply:
Go for it, I just got back from a long vacation so it'll probably be a couple days before I'm back in business :)